packages/styles/scss/patterns/sub-patterns/link-list/_link-list.scss

```diff
@@ -0,0 +1,63 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../../globals/imports';
+@import '../../../globals/utils/hang';
+
+@mixin link-list {
+  .#{$prefix}--link-list {
+    &__heading {
+      margin-top: $carbon--spacing-07;
+      margin-bottom: $carbon--spacing-05;
+
+      @include type-style('expressive-heading-02');
+
+      color: $text-01;
+    }
+    &__CTA {
+      .#{$prefix}--card {
+        &__wrapper {
+          min-height: 116px;
+        }
+        &:not(:hover) {
+          background: $ui-background;
+        }
+        &__heading {
+          @include carbon--type-style('body-short-02');
+
+          color: $link-01;
+        }
+        svg {
+          fill: $interactive-04;
+        }
+      }
+    }
+    &__list {
+      padding-bottom: $carbon--layout-01;
+      @include hang;
+      &__CTA {
+        &:first-child {
+          border-top: 1px solid $ui-03;
+        }
+        &:not(:first-child) {
+          border-top: 1px solid $ui-03;
+        }
+        &:last-child {
+          border-top: 1px solid $ui-03;
+          border-bottom: 1px solid $ui-03;
+        }
+
+        @include carbon--breakpoint('md') {
+          max-width: carbon--rem(640px);
+        }
```

Comment:
Is there a reason to not apply this to the root component? I would also use `mini-units` instead. `carbon--mini-units(80)`.

Reply:
@mkothur it appears you're adding a max-width to `${prefix}--link-list__list__CTA`. Why there, and not `${prefix}--link-list` the highest level class for this pattern?